src/main/java/core/basesyntax/dao/impl/CsvWriteDaoImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.CsvDao;
+import core.basesyntax.dao.WriteDao;
+import core.basesyntax.service.parser.WriteParser;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class CsvWriteDaoImpl extends CsvDao implements WriteDao {
+    private final WriteParser parser;
+
+    public CsvWriteDaoImpl(String destinationPath, WriteParser parser) {
+        super(destinationPath);
+        this.parser = parser;
+    }
+
+    @Override
+    public void save(Map<String, Integer> fruitMap) {
```

Comment:
The similar suggestion. Let`s this method take `List<String>` as parameter. So this class can be not dependent from WriteParser.
This make class to have more compliance to **Single Responsibility** principle, and can be more flexible for the future updates.